Rakefile

```diff
@@ -15,7 +15,7 @@ def collect_task_errors(tasks)
 end
 
 def base_cucumber_path
-  "artifacts/#{ENV['BROWSER']}/#{ENV['BROWSERSTACK'] ? ENV['BROWSERSTACK_CONFIGURATION_OPTIONS'] : 'other'}"
+  "artifacts/#{ENV['BROWSER']}/#{ENV.fetch('BROWSERSTACK', 'false') != 'false' ? ENV['BROWSERSTACK_CONFIGURATION_OPTIONS'] : 'other'}"
```

Comment:
if you just write this as `ENV["BROWSERSTACK"] == "true"` it uses the same check we use in the main site. It's slightly more optimal as it won't do a double eval.